Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -1595,6 +1597,13 @@ def condition(self, fragment, stepList, key, hasHarvest):
         'Era' : 'Phase2C11I13T27M9',
         'ScenToRun' : ['GenSimHLBeamSpot','DigiTrigger','RecoGlobal', 'HARVESTGlobal'],
     },
+    '2026D91' : {
+        'Geom' : 'Extended2026D91',
+        'HLTmenu': '@fake2',
+        'GT' : 'auto:phase2_realistic_T30',
+        'Era' : 'Phase2C11I13M9',
```

Comment:
The era is not done properly here. D91 uses C17, so the era should be `Phase2C17I13M9`. https://github.com/cms-sw/cmssw/blob/master/Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py#L1583-L1589

Reply:
I've addressed the comment here: https://github.com/cms-sw/cmssw/pull/36931.
BTW @adewit the new D91 geometry is not documented in https://github.com/cms-sw/cmssw/blob/master/Configuration/Geometry/README.md, is it intended?

Reply:
Thanks @mmusich - the lack of documentation is not intentional. Can we add it in #36931?

In the list of tracker versions:
`T30: Phase2 tilted tracker, exploratory geometry *only to be used in D91 for now*. Outer Tracker (v8.0.1): based on v8.0.0 with updated TB2S spacing. Inner Tracker (v6.4.0): based on v6.1.5 but TFPX with more realistic module positions`

And at the end of the list of geometries:
`D91 = T30+C17+M10+I15+O9+F6`

Reply:
sure. Done in https://github.com/cms-sw/cmssw/pull/36931/commits/2ae9367c594b54be186d830e3e1f7d00223bdf53